mail_sendgrid_mass_mailing/models/mail_mail.py

```diff
@@ -7,7 +7,7 @@
 _logger = logging.getLogger(__name__)
 
 try:
-    from sendgrid.helpers.mail.mail import TrackingSettings, \
+    from sendgrid.helpers.mail import TrackingSettings, \
         SubscriptionTracking
 except ImportError:
     _logger.error("ImportError raised while loading module.")
```

Comment:
While you're at it, could you replace all the import error logs by warnings? This will prevent some instances to fail when sendgrid is not installed. Please check other cases in the module as well in mail_sendgrid module. Thanks!